packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -246,6 +247,7 @@ export abstract class Chart extends Observable {
 
         const scene: Scene | undefined = resources?.scene;
         const container = resources?.container ?? options.processedOptions.container ?? undefined;
+        const styleContainer = resources?.styleContainer ?? options.processedOptions.styleContainer ?? undefined;
```

Comment:
```suggestion
        const styleContainer = resources?.styleContainer ?? options.processedOptions.styleContainer;
```
nit: `?? undefined` is redundant.

Reply:
Fixed